modules/onboard-azure-eventhub/README.md

```diff
@@ -0,0 +1,68 @@
+# onboard-azure-eventhub
+Creates and onboards an Azure Event Hub to DSF Hub, alongside creating the Storage Account used by DSF to store a marker when pulling data from the Event Hub.
```

Comment:
do we want to mention that the Event Hub connection is using the `default` auth mechanism in this module?